IlmBase/Half/CMakeLists.txt

```diff
@@ -67,11 +67,12 @@ IF (BUILD_ILMBASE_STATIC)
 ENDIF ()
 
 IF (OPENEXR_BUILD_SHARED OR BUILD_ILMBASE_STATIC)
-  INSTALL ( TARGETS
-    ${ILMBASE_LIB_TARGETS}
+  INSTALL ( TARGETS ${ILMBASE_LIB_TARGETS}
+      EXPORT "${TARGETS_EXPORT_NAME}"
       ARCHIVE DESTINATION lib
       LIBRARY DESTINATION lib
       RUNTIME DESTINATION ${RUNTIME_DIR}
+      INCLUDES DESTINATION include/OpenEXR
```

Comment:
This one will modify `INTERFACE_INCLUDE_DIRECTORIES`. I'm not sure about C++ part but usually it's just `include`, e.g.:
- https://cmake.org/cmake/help/latest/manual/cmake-packages.7.html#creating-packages